components/SettingsControls/src/SettingsExpander/SettingsExpander.xaml

```diff
@@ -263,7 +262,7 @@
                         <Border x:Name="ExpanderContentClip"
                                 Grid.Row="1">
                             <Border x:Name="ExpanderContent"
-                                    MinHeight="{TemplateBinding MinHeight}"
+                                    MinHeight="{ThemeResource SettingsExpanderContentMinHeight}"
```

Comment:
Is this the only place `MinHeight` was bound? It's still in the style for the template. If it was overloaded, this is fine I think. Otherwise, we should repurpose it to represent this and keep the template binding, but change the value in the Style to this resource instead of `{StaticResource ExpanderMinHeight}` on L124 right?

Reply:
Yeah, it's been used for both the toggle button part (= header) and the expandable part (= content). I guess that's the reason why this bug appeared (as a SettingsCard in the content normally has a lower minheight because it has less padding).